src/sec_edgar_bulker/core/idx_downloader.py

```
@@ -0,0 +1,470 @@
+        quarter: Optional quarter to download
+        idx_type: Type of index file (e.g., "master", "company", etc.)
+        session: aiohttp session to use
+        config: Configuration object
+        
+    Returns:
+        bool: True if download was successful, False otherwise
+    """
+    downloader = IdxDownloader(config)
+    try:
+        return await downloader._download_idx_file(year, quarter, idx_type, session)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)